galaxy/Chart.yaml

```diff
@@ -22,14 +22,14 @@ dependencies:
       - deploy-s3csi
   - name: galaxy-cvmfs-csi
     repository: https://raw.githubusercontent.com/cloudve/helm-charts/master/
-    version: 2.4.0
+    version: 2.5.0
     condition: cvmfs.deploy
     alias: cvmfs
     tags:
       - deploy-cvmfs
   - name: rabbitmq-cluster-operator
     repository: https://charts.bitnami.com/bitnami
-    version: 2.6.12
+    version: 4.3.22
```

Comment:
It looks like there have been two major version changes. Have these changed the yaml configs in anyway?

Reply:
I don't think so. These are the only notes I was able to find re. the release/upgrade process and nothing there seems to conflict with our setup: https://github.com/bitnami/charts/tree/main/bitnami/rabbitmq-cluster-operator#to-400

I launched 3 clusters with this version and ran a few IWC workflows, all of which worked as expected.